src/main/kotlin/org/rust/lang/core/resolve/ref/RsLitExprReferenceContributor.kt

```diff
@@ -59,7 +59,11 @@ private class RsLiteralFileReferenceSet(
                 val item = parent.ancestorStrict<RsModDeclItem>() ?: parent.ancestorStrict<RsMod>()
                 listOfNotNull(item?.containingMod?.getOwnedDirectory())
             }
-            else -> parentDirectoryContext
+            else -> {
+                val rsElement = element as? RsElement ?: return emptyList()
+                val workspaceRoot = rsElement.containingCargoPackage?.contentRoot ?: return emptyList()
+                return toFileSystemItems(workspaceRoot)
+            }
```

Comment:
Unfortunately, it breaks name resolution and completion in `include` macros and that's why test fails.
For example, the following code is correct
```rust
// -crate_root.txt
/* hello from crate root! */

// -src/sibling.txt
/* hello from src dir! */

// -src/main.rs

use std::fs::File;
use std::io::Read;

fn main() {
    let mut buffer = String::new();
    File::open("crate_root.txt").unwrap().read_to_string(&mut buffer).unwrap();
    println!("{}", buffer);

    let s = include_str!("sibling.txt");
    println!("{}", s);
}
```
and it prints
```
/* hello from crate root! */
/* hello from src dir! */
```
But with these changes, `sibling.txt` cannot be resolved correctly.
So let's use `parentDirectoryContext` for `RsIncludeMacroArgument` as before